l10n_de_holidays/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2018 elego Software Solutions GmbH - Yu Weng
+# Copyright 2018 initOS GmbH - Nikolina Todorova
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": 'Holidays for Germany',
+    "version": '11.0.1.0.0',
+    "license": "AGPL-3",
+    "category": "Human Resources",
+    "author": "Yu Weng <yweng@elegosoft.com>, "
+              "Nikolina Todorova <nikolina.todorova@initos.com>, "
+              "Odoo Community Association (OCA)",
+    "description": """
```

Comment:
Please remove this key, as you already have README

Reply:
@pedrobaeza Thanks for the review. Sorry about delayed correction. I was in holidays last Month.